templates/error/container.md.hbs

```diff
@@ -0,0 +1,15 @@
+# Contents of {{name}}
```

Comment:
Shouldn't this be in the root templates folder?

Reply:
ehm yes